response_operations_ui/views/update_event_date.py

```diff
@@ -89,11 +107,12 @@ def update_event_date_submit(short_name, period, tag):
     """Attempts to create the event, returns None if success or returns an error message upon failure."""
     error_message = collection_exercise_controllers.update_event(
         collection_exercise_id=exercise['id'], tag=tag, timestamp=submitted_dt)
-
     if error_message:
         flash(error_message, 'error')
         return redirect(url_for('collection_exercise_bp.update_event_date',
                                 short_name=short_name, period=period, tag=tag))
-
+    success_message = 'Event date updated.' if tag != 'return_by' else 'Event date updated. Nudge emails that were ' \
```

Comment:
This is kinda ugly.  Probably better to have this either be a long form if/else 
```python
# might have to do success_message = "" here (I forget python variable scoping.. ^^)
if tag != 'return_by':
    success_message = 'Event date updated.' 
else:
    success_message = 'Event date updated. Nudge emails that were scheduled after the return by date will be ' \
                                        'removed. 
    return redirect(url_for('collection_exercise_bp.view_collection_exercise',
                             short_name=short_name, period=period, success_panel=success_message))
```

or do something like 
```python
if tag != 'return_by':
    success_message = 'Event date updated.' 
    return redirect(url_for('collection_exercise_bp.view_collection_exercise',
                             short_name=short_name, period=period, success_panel=success_message))
else:
    success_message = 'Event date updated. Nudge emails that were scheduled after the return by date will be ' \
                                        'removed. 
    return redirect(url_for('collection_exercise_bp.view_collection_exercise',
                             short_name=short_name, period=period, success_panel=success_message))
```

Reply:
ok done!